aws.yaml

```diff
@@ -1,6 +1,6 @@
-- version: 11.1.1
+- version: 11.2.0
```

Comment:
Is it intended to change this version instead of adding a new release? Looks wrong to me now, also because the issue linked in the description talks about `aws-operator@8.2.2` while the operator version here did not change. Do we really have everything in the set tag?

Reply:
The aws-operator release is not done yet. But I guess we can already set it here.

There is no more need for the inactive 11.1.1 release, so we are effectively replacing it.

Reply:
I would then like to also get a new version of `cluster-operator` in because there were at least the go modules changes. Releasing is easy, just creating the Github Release. Should be `2.1.6` then.